README.md

```diff
@@ -527,6 +529,8 @@ Fields listed under `task_definition` correspond to fields that will be included
 
 * `task_size` Contains two fields, CPU and Memory. These fields are required for launching tasks with Fargate launch type. See [the documentation on ECS Task Definition Parameters](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html) for more information.
 
+* `pid` and `ipc` allow you to control the process namespace in which your container runs. Valid values are `task` or `host`. See the [ECS documentation](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#other_task_definition_params) for more information. 
```

Comment:
should we separate these out into 2 bullets? This would be more helpful if it said specifically what each flag effected, vs. lumping them together (even if the task vs. host text and doc link was the same)